common/hashmap.h

```diff
@@ -734,6 +730,11 @@ void HashMap<Key, Val, HashFunc, EqualFunc>::erase(const Key &key) {
 
 /** @} */
 
+template<class Val, class HashFunc, class EqualFunc>
+class HashMap<signed long long, Val, HashFunc, EqualFunc>;
+template<class Val, class HashFunc, class EqualFunc>
+class HashMap<unsigned long long, Val, HashFunc, EqualFunc>;
+
```

Comment:
So, since these do not have an implementation, there should be a comment here that they should not be used

Reply:
What for? Compilation will fall anyway.